aip/general/0158.md

```diff
@@ -159,6 +159,12 @@ now is only getting the first 50 (and does not know to advance pagination).
 Even if the API set a higher default limit, such as 100, the user's collection
 could grow, and _then_ the code would break.
 
+Additionally, [client libraries typically implement automatic
+pagination](../client-libraries/4233.md), typically representing
```

Comment:
```suggestion
Additionally, [client libraries implement automatic
pagination](../client-libraries/4233.md), typically representing
```

Nit: just repeated use of `typically` threw me off - figured I'd just make it an assertion that client libraries do indeed implement pagination :)

Reply:
Done (and fixed a typo in the next line, and then rewrapped.)